source/common/http/async_client_impl.h

```diff
@@ -144,9 +144,9 @@ class AsyncStreamImpl : public AsyncClient::Stream,
     }
     absl::optional<std::chrono::milliseconds> maxInterval() const override { return absl::nullopt; }
 
-    const std::vector<uint32_t> retriable_status_codes_;
-    const std::vector<Http::HeaderMatcherSharedPtr> retriable_headers_;
-    const std::vector<Http::HeaderMatcherSharedPtr> retriable_request_headers_;
+    const std::vector<uint32_t> retriable_status_codes_{};
```

Comment:
It is a bit weird that this change is required here but not in a similar class NullHedgePolicy a few lines above. What toolset are you using to compile envoy?

Reply:
+1 this doesn't make much sense to me and seems like a compiler bug?

/wait